3DAmsterdam/Assets/Amsterdam3D/Scripts/JavascriptInteraction/DrawHTMLOverCanvas.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.Security.AccessControl;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.Video;
+
+namespace Amsterdam3D.JavascriptConnection
+{
+    public class DrawHTMLOverCanvas : MonoBehaviour
+    {
+        [SerializeField]
+        private string htmlObjectID = "";
+
+        private Image image;
+
+        [SerializeField]
+        private bool alignEveryUpdate = false;
+
+        private Rect screenSpaceRectangle;
+
+        private void Awake()
+        {
+            image = GetComponent<Image>();
+        }
+
+        private void Update()
+        {
+            if (alignEveryUpdate)
+                AlignHTMLOverlay();
+        }
+
+        /// <summary>
+        /// Tell JavaScript to make a DOM object with htmlObjectID to align with the Image component
+        /// </summary>
+        private void AlignHTMLOverlay()
```

Comment:
Wat gebeurd hier als het DOM object niet bestaat? Gooit de functie een error, krijg je pas bij javascript een error of blijft het stil.

Reply:
Dan krijg je inderdaad bij je javascript console een null error.
zelfde als wanneer je daar dit uitvoer:
document.getElementById("dezebestaatniet").style.width = "1px";